cmd/test

```diff
@@ -20,19 +20,82 @@ help() {
     echo
 }
 
+basic_help() 
+{
+    echo
+    echo "usage: ${CMD_NAME} ${SUB_CMD_NAME} basic [<options>]"
+    echo
+    printf "    %-2s   %s\n" "" "Options:"
+    printf "    %-3s   %s\n" "" "-w : Execute init --with-pull."
+    printf "    %-3s   %s\n" "" "-f : Force; do not prompt for confirmation."
+    printf "    %-3s   %s\n" "" "-h : Prints this help information."
+    echo
+}
+
 basic() {
+  # Parameters
+  W=0
+  F=0
+  while getopts "wfh" opt; do
+    case $opt in
+      w)
+        W=1
+        ;;
+      f)
+        F=1
+        ;;
+      h)
+        basic_help
+        exit 0
+        ;;
+      *)
+        echo "Invalid parameter(s) or option(s)."
+        basic_help
+        exit 1
+        ;;
+    esac
+  done
+
+  echo "...checking if LDOP is currently running..."
+  PS=$(${CONF_DIR}/adop compose ps -q | wc -l)
```

Comment:
Consider the case where something changes in the compose script as to how it performs standard output which then alters what the correct line count is. Is there a way we can instead parse the State field of the compose command and check for running (or other states where we want to prompt)?